### PR TITLE
docs: update main readme, contributing guide

### PR DESCRIPTION
**Pull request by kon14**
**What kind of change does this PR introduce?** 

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [X] Other

**Does this PR introduce a breaking change?** 

- [ ] Yes
- [X] No

**The PR fulfills these requirements:**

- [X] It's submitted to the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's description  (e.g. `fix #xxx`, where "xxx" is the issue number)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -16,48 +18,51 @@ Welcome! We are really excited that you are interested in contributing to Condui
...
 ```
 
+You may look up supported envs and configuration options for your modules in the [modules section of the documentation](https://getconduit.dev/docs/modules).
+
 ### Committing Changes
 
-Commit messages should follow the [commit message convention](https://github.com/ConduitPlatform/Conduit/blob/master/.github/COMMIT_CONVENTION.md) so that changelogs can be automatically generated.
+Commit messages should follow the [commit message convention](https://github.com/ConduitPlatform/Conduit/blob/main/.github/COMMIT_CONVENTION.md) so that changelogs can be automatically generated.
 
 ### Commonly Used Commands
```
Comment (ChrisPdgn) [inline .github/CONTRIBUTING.md:67]: scope is prefixed with '@conduitplatform'

**Review by ChrisPdgn — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (kon14) [reply .github/CONTRIBUTING.md:67]: Nice catch 👍

**Review by ChrisPdgn — APPROVED** [review]

**Comment by kon14** [comment]
Temporarily drafting this so as replace the dashboard screenshot.

**Review by SotiriaSte — APPROVED** [review]

**Comment by kon14** [comment]
Redrafting to update the CodeFactor badges.

**Review by kkopanidis — APPROVED** [review]

**Pull request merged**